### PR TITLE
[12.0][FIX] sale_financial_risk: Commercial partner not set when value is null

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: sale_financial_risk/hooks.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def create_commercial_partner_id_column(cr):
         SET commercial_partner_id = p.commercial_partner_id
         FROM res_partner p
         WHERE  p.id = sol.order_partner_id and
-                sol.commercial_partner_id <> p.commercial_partner_id;
+                (sol.commercial_partner_id <> p.commercial_partner_id
+                 or sol.commercial_partner_id is null);
```
Comment (pedrobaeza) [inline sale_financial_risk/hooks.py]: This expression covers all:

```suggestion
                sol.commercial_partner_id IS DISTINCT FROM p.commercial_partner_id
```

**Review by pedrobaeza — APPROVED** [review]

File: sale_financial_risk/hooks.py (hunk 1 above)
Comment (carlosdauden) [reply sale_financial_risk/hooks.py]: Changes done

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge patch

Is this needed in upper versions?

**Comment by carlosdauden** [comment]
v13 https://github.com/OCA/credit-control/pull/137

**Pull request merged**